lib/tether/tether.go

```diff
@@ -252,6 +257,41 @@ func (t *tether) setMounts() error {
 			return fmt.Errorf("unsupported volume mount type for %s: %s", k, v.Source.Scheme)
 		}
 	}
+	return t.populateVolumes()
```

Comment:
So this is getting called after we already mount the nfs/device. I am not sure of the behavior of a bind mount after you have already mounted the target. I believe that you will actually end up with the mounted contents of the mounted device. I have not tried it yet though, so I could be wrong.